src/Button/Button.js

```diff
@@ -1,3 +1,4 @@
+import '../../node_modules/fundamental-styles/components/button.scss';
```

Comment:
Does this need to include the `../../node_modules` in the path or could the import be simplified to `import 'fundamental-styles/components/button.scss';`?